wizard/class-instant-articles-wizard-review-submission.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Facebook Instant Articles for WP.
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package default
+ */
+require_once( dirname( __FILE__ ) . '/class-instant-articles-option-fb-app.php' );
+require_once( dirname( __FILE__ ) . '/class-instant-articles-option-fb-page.php' );
+
+use Facebook\InstantArticles\Client\Client;
+
+/**
+ * Class responsible for functionality and rendering of the review settings
+ *
+ * @since 3.1
+ */
+class Instant_Articles_Wizard_Review_Submission {
+
+	const MIN_ARTICLES = 5;
+
+	const STATUS_REJECTED = 'REJECTED';
+	const STATUS_APPROVED = 'APPROVED';
+	const STATUS_PENDING = 'PENDING';
+	const STATUS_NOT_SUBMITTED = 'NOT_SUBMITTED';
+
+	public static function getArticlesForReview() {
+		$post_types = apply_filters( 'instant_articles_post_types', array( 'post' ) );
+		return wp_get_recent_posts(
```

Comment:
`wp_get_recent_posts()` uses `get_posts()` which is an uncached function. `WP_Query` should be used instead here.